src/types/Stream.d.ts

```diff
@@ -0,0 +1,27 @@
+import Client from './Client'
+import { ExprArg } from './query'
+
+export interface Subscription {
+  client: StreamClient
+  dispatcher: EventDispatcher
```

Comment:
This fields don't exist, their `_` prefixed counterparts are private to the subscription class.

Reply:
I think we should document its public interface instead: `on`, `start`, and `close`. Also, the most useful type annotation will probably by the events themselves.